Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp

```diff
@@ -0,0 +1,207 @@
+  auto* utility_layout = new QGridLayout();
+  utility_box->setLayout(utility_layout);
+
+  m_dump_textures = new GraphicsBool(tr("Dump Textures"), Config::GFX_DUMP_TEXTURES);
+  m_load_custom_textures = new GraphicsBool(tr("Load Custom Textures"), Config::GFX_HIRES_TEXTURES);
+  m_prefetch_custom_textures =
+      new GraphicsBool(tr("Prefetch Custom Textures"), Config::GFX_CACHE_HIRES_TEXTURES);
+  m_use_fullres_framedumps = new GraphicsBool(tr("Full Resolution Frame Dumps"),
+                                              Config::GFX_INTERNAL_RESOLUTION_FRAME_DUMPS);
+  m_dump_efb_target = new GraphicsBool(tr("Dump EFB Target"), Config::GFX_DUMP_EFB_TARGET);
+  m_enable_freelook = new GraphicsBool(tr("Free Look"), Config::GFX_FREE_LOOK);
+  m_dump_use_ffv1 = new GraphicsBool(tr("Frame Dumps Use FFV1"), Config::GFX_USE_FFV1);
+
+  utility_layout->addWidget(m_dump_textures, 1, 1);
+  utility_layout->addWidget(m_load_custom_textures, 1, 2);
+  utility_layout->addWidget(m_prefetch_custom_textures, 2, 1);
+  utility_layout->addWidget(m_use_fullres_framedumps, 2, 2);
+  utility_layout->addWidget(m_dump_efb_target, 3, 1);
+  utility_layout->addWidget(m_enable_freelook, 3, 2);
+#if defined(HAVE_FFMPEG)
+  utility_layout->addWidget(m_dump_use_ffv1, 4, 1);
+#endif
+
+  // Misc.
+  auto* misc_box = new QGroupBox(tr("Misc."));
+  auto* misc_layout = new QGridLayout();
+  misc_box->setLayout(misc_layout);
+
+  m_enable_cropping = new GraphicsBool(tr("Crop"), Config::GFX_CROP);
+  m_enable_prog_scan = new QCheckBox(tr("Enable Progressive Scan (Currently broken)"));
+  m_borderless_fullscreen =
+      new GraphicsBool(tr("Borderless Fullscreen"), Config::GFX_BORDERLESS_FULLSCREEN);
+
+  misc_layout->addWidget(m_enable_cropping, 1, 1);
+  misc_layout->addWidget(m_enable_prog_scan, 1, 2);
+#ifdef _WIN32
+  misc_layout->addWidget(m_borderless_fullscreen, 2, 1);
+#endif
+
+  AddWidget(debugging_box);
+  AddWidget(utility_box);
+  AddWidget(misc_box);
+}
+
+void AdvancedWidget::ConnectWidgets()
+{
+  connect(m_enable_prog_scan, &QCheckBox::toggled, this, &AdvancedWidget::SaveSettings);
+  connect(m_load_custom_textures, &QCheckBox::toggled, this, &AdvancedWidget::SaveSettings);
+  connect(m_enable_prog_scan, &QCheckBox::toggled, this, &AdvancedWidget::SaveSettings);
+}
+
+void AdvancedWidget::LoadSettings()
+{
+  m_enable_prog_scan->setChecked(SConfig::GetInstance().bProgressive);
+  m_prefetch_custom_textures->setEnabled(Config::Get(Config::GFX_HIRES_TEXTURES));
+}
+
+void AdvancedWidget::SaveSettings()
+{
+  SConfig::GetInstance().bProgressive = m_enable_prog_scan->isChecked();
+  const auto hires_enabled = Config::Get(Config::GFX_HIRES_TEXTURES);
+  m_prefetch_custom_textures->setEnabled(hires_enabled);
+  if (!hires_enabled)
+    m_prefetch_custom_textures->setEnabled(false);
+}
+
+void AdvancedWidget::OnBackendChanged()
+{
+  const auto supports_fr_framedumps = g_Config.backend_info.bSupportsInternalResolutionFrameDumps;
```

Comment:
Nope. It being checked and it being supported are two entirely different things.